nixos/doc/manual/from_md/configuration/sshfs-file-systems.section.xml

```diff
@@ -0,0 +1,137 @@
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="sec-sshfs-file-systems">
+  <title>SSHFS File Systems</title>
+  <para>
+    <link xlink:href="sshfs">SSHFS</link> is a
+    <link xlink:href="fuse">FUSE</link> filesystem that allows easy
```

Comment:
Sorry, I guess the style of Markdown links where you define the href later is not working with our Markdown to Docbook translation. Can you use the normal link style?

Reply:
Actually no, that isn't it: https://github.com/NixOS/nixpkgs/pull/127783/checks?check_run_id=2887715376#step:4:117 You just need to run the conversion again.

Reply:
Fixed in the version I just force pushed.